proposals/lip-0052.md

```diff
@@ -197,8 +197,6 @@ The following constants are used throughout the document:
 | `ChainID` | bytes | Must be of length `LENGTH_CHAIN_ID`. | Used for chain identifiers. |
 | `CollectionID` | bytes | Must be of length `LENGTH_COLLECTION_ID`. | Used for NFT collection identifiers. |
 | `AttributesArray` | (Module &#124; bytes)[] | Two-dimensional array consisting of `Module` names and corresponding `attributes`. | Used for array of NFT attributes. |
-| `Transaction` | object | Must follow the `transactionSchema` schema defined in the [LIP 0068][lip-0068] with the only difference that `params` property is not serialized and contains the values of parameters of `paramsSchema` for the corresponding transaction. | An object representing a non-serialized transaction. |
-| `CCM` | object | Must follow the `crossChainMessageSchema` schema defined in [LIP 0049][lip-0049#ccmschema]. | The type of cross-chain messages. |
```

Comment:
Also here we my consider to keep the definition

Reply:
As we just have one type `CCM` that is the same everywhere and I don't see any risk of confusing about it, I would prefer to avoid this redundancy in LIPs.